docs/guides/migration-0.3.x.md

```diff
@@ -1,7 +1,55 @@
 # Migration from 0.2.x to 0.3.x
 
-Certain resources undergone changes in order to ensure consistency with 
+Certain resources undergone changes in order to ensure consistency with REST API and standard expecte Terraform behavior.
+
+## provider
+
+* Rewrite `basic_auth` block with `username` and `password` fields, as specified in [main document](https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs#authenticating-with-hostname-username-and-password).
+* Rewrite `azure_auth` block with appropriate [Azure configuration](https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs#special-configurations-for-azure).
+
+## databricks_job
+
+* Rewrite `spark_submit_parameters` with [spark_submit_task](https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/resources/job#spark_submit_task-configuration-block) configuration block.
+* Rewrite `python_file` and `python_parameters` with [spark_python_task](https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/resources/job#spark_python_task-configuration-block) configuration block.
+* Rewrite `jar_uri`, `jar_main_class_name`, and `jar_parameters` with [spark_jar_task](https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/resources/job#spark_jar_task-configuration-block) configuration block.
+* Rewrite `notebook_path` and `notebook_base_parameters` with [notebook_task](https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/resources/job#notebook_task-configuration-block) configuration block.
+* Rewrite `library_jar`, `library_egg`, `library_whl`, `library_pypi`, `library_cran`, and `library_maven` with [library](https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/resources/cluster#library-configuration-block) configuration block.
+
+## databricks_notebook
+
+* Rename `content` to `content_base64`, as this closer represents actual data within the field and simplifies internal code reusability.
+* Remove `format` attribute. Starting from v0.3.0 it behaves as if it is set to `SOURCE`.
+* Remove `overwrite` attribute. Starting from v0.3.0 it behaves as if it is set to `true`.
+* Remove `mkdirs` attributes. Starting from v0.3.0 it behaves as if it is set to `true`.
+
+## databricks_cluster
+
+* Rewrite `library_jar`, `library_egg`, `library_whl`, `library_pypi`, `library_cran`, and `library_maven` with [library](https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/resources/cluster#library-configuration-block) configuration block.
+
+## databricks_dbfs_file
+
+* Rename `content` to `content_base64`, as this closer represents actual data within the field and simplifies internal code reusability.
```

Comment:
Similarly to notebook write about  overwrite, mkdirs fields. And also about content_b64_md5, validate_remote_file fields